cognite/v05/timeseries.py

```diff
@@ -150,8 +151,8 @@ def _get_datapoints_helper(timeseries, aggregates=None, granularity=None, start=
         list of datapoints: A list containing datapoint dicts.
     '''
     api_key, project = kwargs.get('api_key'), kwargs.get('project')
-    timeseries = timeseries.replace('/', '%2F')
-    url = config.get_base_url(api_version=0.5) + '/projects/{}/timeseries/data/{}'.format(project, timeseries)
+    name = name.replace('/', '%2F')
```

Comment:
Is there some sort of urlencode-function you could use here instead? and elsewhere

Reply:
Yes, there is. Will use.